doc/crossvalidation.ipynb

```diff
@@ -247,7 +247,7 @@
SZIkSVLtcCqvJEmSJKlUFqaSJEmSpFJZmEqSJEmSSmVhKkmSJEkqlYWpJEmSJKlUFqaSJEmSpFJZmEqSJEmSSmVhKkmSJEkqlYWpJEmSJKlUFqaSJEmSpFJZmEqSJEmSSmVhKkmSJEkqlYWpJEmSJKlUFqaSJEmSpFJZmEqSJEmSSmVhKkmSJEkqVaGFaURMi4h/iYjfRMRlRfYlSZIkSapOhRWmETEA+A7waeBI4IKIOLKo/iRJkiRJ1WlggW0fB/wmM1cDRMTfAZ8Bnimwz8Ks+/1rvPHWO/zHHz1WdiiS+ojxo4cz5/SGssMoVvNc2Lyk9/utmwCT5/V+v5IkqRRFTuU9GHihzfu1lX3vERGzI6IpIppaWloKDEeSJEmS1BcVOWLaLZm5AFgA0NjYmCWH06GDRgwFYNaff6zkSCSpFzlqKUmSekGRI6YvAoe0eT+msk+SJEmSpB2KLEyfBA6LiHERsRdwPnB3gf1JkiRJkqpQYVN5M/PtiPgrYBEwALgpM5cX1Z8kSZIkqToV+oxpZt4L3FtkH5IkSZKk6lbkVF5JkiRJkrpkYSpJkiRJKpWFqSRJkiSpVBamkiRJkqRSRWaWHcMOEdEC/LbsODoxCthYdhDqMeaztpjP2mI+a4v5rC3ms7aYz9rS1/N5aGZ+sL0Dfaow7esioikzG8uOQz3DfNYW81lbzGdtMZ+1xXzWFvNZW6o5n07llSRJkiSVysJUkiRJklQqC9Nds6DsANSjzGdtMZ+1xXzWFvNZW8xnbTGftaVq8+kzppIkSZKkUjliKkmSJEkqlYWpJEmSJKlUVV2YRsQhEbE4Ip6JiOUR8eXK/hER8WBErKr8rKvsPyIiHouINyPiqzu19ZVKG8siYmFEDO6gzy9U2l0VEV9os//qiHghIl7tIubJEfGriPhNRFwfEVHZf16l/3cjoiqXeN5TNZbPKyPixYhYUnmdsaf3p9rUWD6PrcT2q4i4JyKG7+n9qTZVms92z4uIv6zkcklEPBoRR+7ufalWNZbP/9Xmd+2vI2LL7t6XalVt+YyIIRHxfyNiZaWva9oc+2REPBURb0fEuXt6b6pRjeVzVkS0tPlv9OI9vT/VpsbyeWhE/GNELI2IhyNizJ7en/fIzKp9AQcCkyrbw4BfA0cCfwNcVtl/GfA/K9v7Ax8Brga+2qadg4HngA9U3t8GzGqnvxHA6srPusp2XeXYlEo8r3YR8z9Xzg3gPuDTlf31wOHAw0Bj2ffWfO5xPq9sG1N/fNVYPp8ETqhsfxH4b2XfX/PZrXy2ex4wvM32dOD+su+v+dz9fO50zpeAm8q+v+azyzwNAU6qbO8F/Jw//r4dCxwD/Ag4t+x7az73OJ+zgL8t+56azx7L598DX6hsnwzc0pP3qqpHTDPzpcx8qrL9CrCC1qR9Bvhh5bQfAp+tnLMhM58E3mqnuYHAByJiIK0JWdfOOacDD2bm7zNzM/AgMK3S9uOZ+VJn8UbEgbT+QfR4tmb0R21iW5GZ/9L9T197aimfqrl8/ivgkcr2g8A5XX3+WlNt+ezsvMzc2ubtUKDfrQJYS/ncyQXAwq7aqjXVls/MfD0zF1e2/wA8BYypvF+TmUuBd7v7+WtNLeVTNZfPI4H/V9leXPkMPaaqC9O2ImIsMBF4Ahjd5qb/Dhjd2bWZ+SJwLfA88BLwcmY+0M6pBwMvtHm/trKvuw6uXLO71/cbNZLPv6pMdbhp+/SM/qoG8rmcP/7yPQ84ZBfarTlVks9ORcS/i4hnaf0X60t7qt1qVAv5hNYpZsA4/vhHU79UbfmMiP2AM4F/3J3ra12N5POcyt9Dt0eE//+s7nz+Eji7sn0WMCwiRu5O2+2picI0IvYB7gDm7vQv4VRGPjr91/BK0fAZWv+HdhAwNCIuLChcdaFG8jkf+DAwgdZfHt/q5f77jBrJ5xeBfxsRzbROw/lDL/ffZ9RIPsnM72Tmh4H/DPx1b/ffV9RKPivOB27PzHdK6r901ZbPyqjPQuD6zFxdVD/VqkbyeQ8wNjOPoXXk7ocdXV/raiSfXwVOiIingROAF4Ee+51b9YVpRAyiNcm3ZuZPK7vXV6blbZ+et6GLZj4FPJeZLZn5FvBT4OMR8dE2D2tPp/Xmt/2XnjGVfR3FNqDN9d+onNt2akOn1/dHtZLPzFyfme9k5rvAjcBx3bsDtaWG8rkyM0/LzMm0/pJ+tnt3oLZUWT676+/op1PwazCf59MPp/FuV6X5XACsysx5u/JZ+4NayWdmbsrMNytvvwdM7uqz16Iayue6zDw7MycC/6Wyr8cWnBvYUw2VISIC+D6wIjOva3PobuALwDWVnz/roqnngSkRMQTYBpwCNGXmE7SOeG3vbwTw3+OP0zJPAy7vqNHKv9pOaLsvIrZGxBRah/D/HPjfXX3O/qKW8hkRB7aZnnEWsKyLmGtOjeVz/8zcEBF/Quvo2ne7iLnmVGM+O/ksh2XmqsrbPwVWdXZ+LaqlfFbaP4LWRT4e6+41taQa8xkR3wT2BfrdKq1dqaV87vT30HRan6/sV2osn6OA31cGXi4Hbuoi5l2TfWC1qt19AZ+gddh7KbCk8joDGEnrXOhVwEPAiMr5B9A6z3orsKWyPbxy7CpgJa0FxC3A3h30+UXgN5XXRW32/02lvXcrP6/s4PrGSh/PAn8LRGX/WZXr3gTWA4vKvr/mc4/yeQvwq8pnuRs4sOz7az73KJ9fpnUVvV/T+j+QKPv+ms9u5bPd84Bv0/rc8BJaF29oKPv+ms/dz2fl2JXANWXfV/PZvXzSOoKTtBYp2+O9uHLsI5XrXgM2AcvLvr/mc4/y+T9o/X37S1p/3x5R9v01n3uUz3Mr8f6a1hHwdvvf3df2P7okSZIkSSpF1T9jKkmSJEmqbhamkiRJkqRSWZhKkiRJkkplYSpJkiRJKpWFqSRJkiSpVBamkiRJkqRSWZhKkiRJkkr1/wEuNaFP79DgzgAAAABJRU5ErkJggg==\n",
```

Comment:
I guess Jupyter changed how images are stored (`\n` at the end). This line change seems unnecessary, but we'll have this with every new update to the notebook. @koaning are you okay with updating this ipynb to the new style in this PR?

Reply:
That's annoying.

But no biggy. Feel free to update.